CHANGELOG.md

```diff
@@ -9,6 +9,7 @@ All notable changes to this project will be documented in this file.
 -   Python support for a simple class.
 -   #1430 : Added conjugate support to booleans.
 -   #1452 : Added C printing support for a class containing only functions.
+-   #1472 : Added C printing support for a class containing only scalar data.
```

Comment:
Does the "only" mean that we have an either/or situation? Or is it possible to have a class with both scalar data and functions after this PR and #1452